docs/building.md

```diff
@@ -33,6 +33,16 @@ You can use brew as shown below
 brew install bdw-gc
 ```
 
+### OS X Yosemite
+
+OS X Yosemite uses outdated version of `clang` so you might have to install a newer version via brew:
+
+```
+brew install homebrew/versions/llvm37
```

Comment:
I wasn't sure whether I should go with "latest available" or "earliest necessary" and went for the latter. I assumed that people would expect the check (which checks for 3.7) and the documentation to align.